packages/@uppy/transloadit/src/AssemblyWatcher.ts

```diff
@@ -49,38 +49,41 @@ class TransloaditAssemblyWatcher<
   }
 
   #onAssemblyComplete = (assembly: AssemblyResponse) => {
-    if (!this.#watching(assembly.assembly_id)) {
+    const assemblyId = assembly.assembly_id
+    if (!assemblyId || !this.#watching(assemblyId)) {
```

Comment:
Again, I don't think asssembly_id can ever be undefined? Or am I mistaken and do the new types say this?